src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Specifications/CatalogProductIsInStockSpecification.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.InventoryModule.Core.Model;
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.XDigitalCatalog.Specifications
+{
+    public class CatalogProductIsInStockSpecification
+    {
+        public virtual bool IsSatisfiedBy(CatalogProduct catalogProduct, IEnumerable<InventoryInfo> InventoryAll)
```

Comment:
Please use formal argument `ExpProduct` it contains all you need include Inventory